sqlalchemy_cockroachdb/base.py

```diff
@@ -191,39 +189,40 @@ def has_table(self, conn, table, schema=None):
     # The upstream implementations of the reflection functions below depend on
     # correlated subqueries which are not yet supported.
     def get_columns(self, conn, table_name, schema=None, **kw):
+        _include_hidden = kw.get("include_hidden", False)
         if not self._is_v2plus:
```

Comment:
perhaps we should remove this `if not self._is_v2plus:` branch. the line below that accesses `row.is_hidden` will break since there won't be any is_hidden column

we no longer support versions that old anyway.